code/obj/item/device/pda2/bot_control.dm

```diff
@@ -87,40 +86,25 @@
 					. += "No bots found.<BR>"
 
 				else
-					for(var/obj/machinery/bot/secbot/B in src.botlist)
+					for(var/mob/living/critter/robotic/securitron/B in src.botlist)
 						. += "[B] at [get_area(B)]</A><BR>"
-						. += "Mode: "
-						switch(B.mode)
-							if(0)
```

Comment:
no mode?

Reply:
I realized after writing the entire AI that this feature existed, and have no clue on how to reasonably rewrite it to support "what am I doing" reports